script.js

```diff
@@ -1,4 +1,154 @@
+  if (
+    (input == "scissors" && type == "paper") ||
+    (input == "paper" && type == "stone") ||
+    (input == "stone" && type == "scissors")
+  ) {
+    userWins += 1;
+    totalTimes = userWins + computerWins;
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> You win! <br> <br>
+   So far ${userName}, You've won ${userWins}/${totalTimes} turns. Good on you! 
+   
+  <br><br>  Now you can type "scissors" "paper" or "stone" to play another round or change game mode!`;
+  }
+  if (input == type) {
+    console.log("Capture");
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> It's a draw ${userName}! Try again. <br> <br> Now you can type "scissors" "paper" or "stone" to play another round!`;
+  } else {
+    computerWins += 1;
+    totalTimes = userWins + computerWins;
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> You lose! Bummer. <br> <br> 
+     So far ${userName}, you've been winning ${userWins}/${totalTimes} turns. Pretty good!`;
+  }
+};
+var reversedSps = function (type, input) {
+  var totalTimes = 0;
+  //Enter scissors paper stone
+  if (!(input == "scissors" || input == "paper" || input == "stone")) {
+    return `Please Enter scissors, paper or stone`;
+  }
+  if (
+    (input == "paper" && type == "scissors") ||
+    (input == "stone" && type == "paper") ||
+    (input == "scissors" && type == "stone")
+  ) {
+    userWins += 1;
+    totalTimes = userWins + computerWins;
+    console.log(input);
+    console.log(type);
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> You win! <br> <br>
+   So far${userName}, You've win ${userWins}/${totalTimes} turns. Good on you! 
+   
+   <br> Now you can type "scissors" "paper" or "stone" to play another round or change!`;
+  }
+  if (input == type) {
+    console.log("Capture");
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> It's a draw ${userName}! Try again. <br> <br> Now you can type "scissors" "paper" or "stone" to play another round!`;
+  } else {
+    computerWins += 1;
+    totalTimes = userWins + computerWins;
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> You lose! Bummer. <br> <br> 
+     So far ${userName}, you've been winning ${userWins}/${totalTimes} turns. Pretty good!`;
+  }
+};
+
+var koreanSps = function (type, input) {
+  var totalTimes = 0;
+  //Enter scissors paper stone
+  if (!(input == "scissors" || input == "paper" || input == "stone")) {
+    return `Please Enter scissors, paper or stone`;
+  }
+  //Gameplay
+  if (
+    (input == "scissors" && type == "paper") ||
+    (input == "paper" && type == "stone") ||
+    (input == "stone" && type == "scissors")
+  ) {
+    userWins += 1;
+    totalTimes = userWins + computerWins;
+    return `The computer chose ${type}. <br> You chose ${input}. <br> <br> You win! <br> <br>
+   So far ${userName}, You've won ${userWins}/${totalTimes} turns. Good on you! 
```

Comment:
As this win/loss/draw condition is repeated for the normalSPS, might want to consider making these reuseable functions

Reply:
I was thinking of doing that too. Wanted to create another function to return the win/loss/draw statements but I didn't know how to because there the user wins and total wins to keep track of as well.

Reply:
perhaps the sps game could be broken down further. One function for just determinining who wins at sps mini-game and the other for the string output.

Then logically you can order it as

var whoWins=spsMinigame()      //gets who wins
if( whoWins=="computer")
{
//increment computer wins, number of total games
//...
}

var outputString= resultStrings();

Just thinking. What do you think? ^^